tests/Application/templates/bundles/SyliusShopBundle/Cart/Summary/_totals.html.twig

```diff
@@ -0,0 +1,65 @@
+{% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
+
+{% set items_subtotal = sylius_order_items_subtotal(cart) %}
+{% set tax_included = sylius_order_tax_included(cart) %}
+{% set tax_excluded = sylius_order_tax_excluded(cart) %}
+
+<div class="ui segment">
+    <h2 class="ui dividing header">{{ 'sylius.ui.summary'|trans }}</h2>
+
+    {{ sylius_template_event('sylius.shop.cart.summary.totals', {'cart': cart}) }}
+
+    <table class="ui very basic table">
+        <tbody>
+        <tr>
+            <td>{{ 'sylius.ui.items_total'|trans }}:</td>
+            <td class="right aligned" {{ sylius_test_html_attribute('cart-items-total') }}>{{ money.convertAndFormat(items_subtotal) }}</td>
+        </tr>
+        {% if cart.orderPromotionTotal %}
+            <tr>
+                <td>{{ 'sylius.ui.discount'|trans }}:</td>
+                <td id="sylius-cart-promotion-total" {{ sylius_test_html_attribute('cart-promotion-total') }} class="right aligned">{{ money.convertAndFormat(cart.orderPromotionTotal) }}</td>
+            </tr>
+        {% endif %}
+        {% if cart.shipments is not empty %}
+            <tr>
+                <td>{{ 'sylius.ui.shipping_estimated_cost'|trans }}:</td>
+                <td class="right aligned">
+                    {% if cart.getAdjustmentsTotal('shipping') > cart.shippingTotal %}
+                        <span class="old-price">{{ money.convertAndFormat(cart.getAdjustmentsTotal('shipping')) }}</span>
+                    {% endif %}
+                    <span id="sylius-cart-shipping-total" {{ sylius_test_html_attribute('cart-shipping-total') }}>{{ money.convertAndFormat(cart.shippingTotal) }}</span>
+                </td>
+            </tr>
+        {% endif %}
+        <tr {% if tax_included and not tax_excluded %}class="tax-disabled"{% endif %}>
```

Comment:
Hint: you can break these expressions to multi-line; `{%-` keeps no wrapping whitespace before and `-%}` after. FYI. :)